crates/editor/src/editor.rs

```diff
@@ -328,6 +328,7 @@ enum DisplayDiffHunk {
     },
 }
 
+#[derive(Copy, Clone)]
```

Comment:
Sorry if I missed anything, but do we get rid of copy and clone if we don’t need it?